test/integration/scheduler/queue_test.go

```diff
@@ -501,6 +500,60 @@ func TestCoreResourceEnqueue(t *testing.T) {
 			wantRequeuedPods:          sets.New("pod1"),
 			enableSchedulingQueueHint: []bool{true},
 		},
+		{
+			name:         "Pod rejected by the interPodAffinity plugin is requeued when updating the label of the node to make it match the pod anti affinity",
+			initialNodes: []*v1.Node{st.MakeNode().Name("fake-node").Label("node", "fake-node").Obj()},
+			pods: []*v1.Pod{
+				// - pod1 and pod2 will be rejected by the PodAntiAffinity plugin.
+				st.MakePod().Name("pod1").PodAntiAffinityExists("bbb", "zone", st.PodAntiAffinityWithRequiredReq).Container("image").Obj(),
+				st.MakePod().Name("pod2").PodAntiAffinityExists("ccc", "region", st.PodAntiAffinityWithRequiredReq).Container("image").Obj(),
```

Comment:
Those pods are schedulable from the first step and the test failed.
You have to create a scheduled Pod in `initialPods` that the `pods` hate.